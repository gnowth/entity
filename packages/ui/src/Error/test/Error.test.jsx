import 'jest-styled-components';
import renderer from 'react-test-renderer';
import React from 'react';
import { IntlProvider } from 'react-intl';

import UIError from '../Error';

it('renders correctly', () => {
  const tree = renderer.create(
    <IntlProvider locale="en" messages={{}}>
      <UIError>Error</UIError>
    </IntlProvider>,
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
