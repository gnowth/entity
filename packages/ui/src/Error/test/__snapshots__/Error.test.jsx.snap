// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  color: black;
}

@media print {
  .c0 {
    color: black;
  }
}

<span
  className="c0"
>
  Error
</span>
`;
